content/client-lib-development-guide/features.textile

```diff
@@ -232,6 +232,22 @@ h3(#rest-auth). Auth
 ** @(RSA16b)@ If the library is provided with a token without the corresponding @TokenDetails@, then this holds a @TokenDetails@ instance in which only the @token@ attribute is populated with that token string
 ** @(RSA16c)@ Is set with the current token (if applicable) on instantiation and each time it is replaced, whether the result of an explicit @Auth#authorize@ operation, or a library-initiated renewal resulting from expiry or a token error response
 ** @(RSA16d)@ Is empty if there is no current token, including after a previous token has been determined to be invalid or expired, or if the library is using basic auth
+* @(RSA17)@ @Auth#revokeTokens@ function: This function is intended to be used to revoke tokens using provided @TokenRevocationRequest@ details. It internally creates request for  "REST revokeTokens endpoint":/rest-api/#revokeTokens and handles the response.
```

Comment:
Am I doing something wrong, I cannot see the REST API documentation for this, in either https://ably.com/docs/rest-api#revoke-tokens or https://docs.ably.io/rest-api/#revoke-tokens.  Where is that documentation?  Should it not exist in the REST API documentation?

Reply:
@mattheworiordan Sorry I didn't know that was supposed to turn into a link. I made it so it goes to the full path 3127a70c431000bbfd6e052768f048011949fa91